src/qt/guiutil.cpp

```diff
@@ -755,12 +755,8 @@ QString formatServicesStr(quint64 mask)
 {
     QStringList strList;
 
-    for (int i = 0; i < 64; i++) {
-        uint64_t check = 1ull << i;
-        if (mask & check)
-        {
-            strList.append(QString::fromStdString(serviceFlagToStr(mask, i)));
-        }
+    for (const auto& flag : serviceFlagsToStr((uint64_t)mask)) {
```

Comment:
```suggestion
    for (const auto& flag : serviceFlagsToStr(uint64_t{mask})) {
```

nit: this is slightly preferable (assuming it compiles)

Reply:
Thanks for the hint. I wasn't aware that

```cpp
double a = 1.2;
uint64_t b;
b = uint64_t{a};
```

wouldn't compile due to type narrowing (whereas `b = (uint64_t)a` would compile).

In our case however the typecast is not needed, so I have removed it altogether.